src/app/fdctl/run/tiles/fd_resolv.c

```diff
@@ -133,6 +199,27 @@ after_frag( fd_resolv_ctx_t *   ctx,
   uchar    const * payload = dcache_entry;
   fd_txn_t const * txn     = (fd_txn_t const *)( dcache_entry + fd_ulong_align_up( payload_sz, 2UL ) );
 
+  /* If we can't find the recent blockhash ... it means one of two things,
+  
+     (1) It's really old (more than 28 minutes) or just non-existent.
+     (2) It's really new (we haven't seen the bank yet).
```

Comment:
Or it's a durable nonce transaction, but the logic for this is still right. Just update the comment.